{"uid":"15c20f744a97ef49","name":"test_copy_read_only_file_with_xattr","fullName":"tests.test_api_build#test_copy_read_only_file_with_xattr","historyId":"8f800bd25b9f9b621339dcc17d2130a9","time":{"start":1673024145262,"stop":1673024145265,"duration":3},"status":"skipped","statusMessage":"XFAIL reason: setfattr not possible in /home/runner/test_copy_read_only_file_with_0/serial/_temp/pytest.conda-build, see https://stackoverflow.com/a/46598063\n\n_pytest.outcomes.XFailed: setfattr not possible in /home/runner/test_copy_read_only_file_with_0/serial/_temp/pytest.conda-build, see https://stackoverflow.com/a/46598063","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/conda-build/conda-build/tests/test_api_build.py\", line 1110, in test_copy_read_only_file_with_xattr\n    subprocess.check_call(f'setfattr -n user.attrib -v somevalue {ro_file}', shell=True)\n  File \"/usr/share/miniconda/envs/test/lib/python3.10/subprocess.py\", line 369, in check_call\n    raise CalledProcessError(retcode, cmd)\nsubprocess.CalledProcessError: Command 'setfattr -n user.attrib -v somevalue /home/runner/test_copy_read_only_file_with_0/serial/_temp/pytest.conda-build/_xattr_copy/mode_400_file' returned non-zero exit status 127.\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/conda-build/conda-build/tests/test_api_build.py\", line 1112, in test_copy_read_only_file_with_xattr\n    return pytest.xfail(\"setfattr not possible in {}, see https://stackoverflow.com/a/46598063\".format(\n  File \"/usr/share/miniconda/envs/test/lib/python3.10/site-packages/_pytest/outcomes.py\", line 252, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: setfattr not possible in /home/runner/test_copy_read_only_file_with_0/serial/_temp/pytest.conda-build, see https://stackoverflow.com/a/46598063","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1673022690836,"stop":1673022690836,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"tmp_path","time":{"start":1673024145259,"stop":1673024145260,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"tmpdir","time":{"start":1673024145260,"stop":1673024145260,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_workdir","time":{"start":1673024145260,"stop":1673024145260,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"monkeypatch","time":{"start":1673024145261,"stop":1673024145261,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"default_testing_config","time":{"start":1673024145261,"stop":1673024145261,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_config","time":{"start":1673024145261,"stop":1673024145261,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_homedir","time":{"start":1673024145261,"stop":1673024145262,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL reason: setfattr not possible in /home/runner/test_copy_read_only_file_with_0/serial/_temp/pytest.conda-build, see https://stackoverflow.com/a/46598063\n\n_pytest.outcomes.XFailed: setfattr not possible in /home/runner/test_copy_read_only_file_with_0/serial/_temp/pytest.conda-build, see https://stackoverflow.com/a/46598063","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/conda-build/conda-build/tests/test_api_build.py\", line 1110, in test_copy_read_only_file_with_xattr\n    subprocess.check_call(f'setfattr -n user.attrib -v somevalue {ro_file}', shell=True)\n  File \"/usr/share/miniconda/envs/test/lib/python3.10/subprocess.py\", line 369, in check_call\n    raise CalledProcessError(retcode, cmd)\nsubprocess.CalledProcessError: Command 'setfattr -n user.attrib -v somevalue /home/runner/test_copy_read_only_file_with_0/serial/_temp/pytest.conda-build/_xattr_copy/mode_400_file' returned non-zero exit status 127.\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/conda-build/conda-build/tests/test_api_build.py\", line 1112, in test_copy_read_only_file_with_xattr\n    return pytest.xfail(\"setfattr not possible in {}, see https://stackoverflow.com/a/46598063\".format(\n  File \"/usr/share/miniconda/envs/test/lib/python3.10/site-packages/_pytest/outcomes.py\", line 252, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: setfattr not possible in /home/runner/test_copy_read_only_file_with_0/serial/_temp/pytest.conda-build, see https://stackoverflow.com/a/46598063","steps":[],"attachments":[{"uid":"e63f5fa9480b122c","name":"stderr","source":"e63f5fa9480b122c.txt","type":"text/plain","size":32}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"monkeypatch::0","time":{"start":1673024145268,"stop":1673024145268,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_homedir::return_to_saved_path","time":{"start":1673024145268,"stop":1673024145268,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_workdir::return_to_saved_path","time":{"start":1673024145269,"stop":1673024145269,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.skipif(False, reason='xattr code written here is specific to linux')"},{"name":"tag","value":"serial"},{"name":"tag","value":"sanity"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_api_build"},{"name":"host","value":"fv-az248-247"},{"name":"thread","value":"3770-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_api_build"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@pytest.mark.skipif(False, reason='xattr code written here is specific to linux')","serial","sanity"]},"source":"15c20f744a97ef49.json","parameterValues":[]}